doc/doxygen/src/getting-started.md

```diff
@@ -39,7 +39,7 @@ simplest way to compile and link an application with RIOT, is to set up a
 Makefile providing at least the following variables:
 
  * `APPLICATION`: should contain the (unique) name of your application
- * `BOARD`: specifies the platform the application should be build for by
+ * `BOARD`: specifies the platform the application should be built for by
```

Comment:
This was also fixed in https://github.com/RIOT-OS/RIOT/pull/10113 so you might not have rebased properly.